contain_learning/Dockerfile

```
@@ -0,0 +1,5 @@
+FROM centos:latest
+RUN yum install python36 -y
+RUN pip3 install numpy
+RUN pip3 install keras
+RUN pip3 install sklearn 
```

Identified a potential vulnerability, here’s a suggested change for The Dockerfile does not specify a fixed version for the base image or the installed packages, which can lead to unpredictable builds if the latest version changes.:

```suggestion
FROM centos:7.9.2009
RUN yum install python36-3.6.8 -y
RUN pip3 install numpy==1.21.0
RUN pip3 install keras==2.4.3
RUN pip3 install sklearn==0.24.2
```